security/protect-hosts.md

```diff
@@ -182,6 +184,52 @@ To change this parameter for all hosts, edit the **FelixConfiguration** object n
    calicoctl apply -f default-felix-config.yaml
    ```
 
+#### Automatically create host endpoints for Kubernetes nodes
+
+{{site.prodname}} can create and manage the lifecycle of host endpoints for your Kubernetes cluster. You can enable this behavior using **KubeControllersConfiguration**.
+
+> **Important**: Before enabling this feature, ensure that [failsafe ports](#avoid-accidentally-cutting-all-host-connectivity) have been reviewed
+> and that the appropriate global network policy is in place.
+>
+> If host endpoints are added and network policy is not in place, the {{site.prodname}} default is to deny traffic to/from the endpoints (except for traffic allowed by failsafe rules).
+{: .alert .alert-danger}
+
+1. Get a copy of the object to edit.
+
+   ```bash
+   calicoctl get kubecontrollersconfiguration default --export -o yaml > default-kube-config.yaml
+   ```
+1. Open the file in a text editor and add the parameter, **autoCreate** to **spec.controllers.node**. For example:
+
+   ```yaml
+   apiVersion: projectcalico.org/v3
+   kind: KubeControllersConfiguration
+   metadata:
+     name: default
+   spec:
+     logSeverityScreen: Info
+     healthChecks: Enabled
+     etcdv3CompactionPeriod: 10m
+     controllers:
+       node:
+         reconcilerPeriod: 5m
+         syncLabels: Enabled
+         hostEndpoint:
+           autoCreate: Enabled
+       policy:
+         reconcilerPeriod: 5m
+       workloadEndpoint:
+         reconcilerPeriod: 5m
+       serviceAccount:
+         reconcilerPeriod: 5m
+       namespace:
+         reconcilerPeriod: 5m
+   ```
+
+1. Update the KubeControllersConfiguration on the cluster.
+   ```bash
+   calicoctl apply -f default-felix-config.yaml
+
```

Comment:
Are you adding an example of "an appropriate global policy" in the new How to? Or is this the only place to expose this new feature. If just this doc, consider adding an example of a global network policy just to connect all of the dots.

Reply:
The section referred to is gone now